neurokit2/complexity/complexity_optimize.py

```diff
@@ -273,7 +273,7 @@ def _complexity_r(signal, delay=None, dimension=None):
     r_range = modulator * np.std(signal, ddof=1)
     ApEn = np.zeros_like(r_range)
     for i, r in enumerate(r_range):
-        ApEn[i] = entropy_approximate(signal, delay=delay, dimension=dimension, r=r_range[i])["ApEn"]
+        ApEn[i] = entropy_approximate(signal, delay=delay, dimension=dimension, r=r_range[i])[0]
```

Comment:
that's dangerous we should avoid querying keys from dictionaries by index because indices (i.e., the order of keys) can change - it's best to query by name

Reply:
@DominiqueMakowski this isn't a key from the dictionary though, its the entropy value itself since the output is a tuple 😄 if extracting a value from a dict key I'll instead put something like `[1]['embedding_dimension']`

Reply:
ah yes my bad no so then it's fine a tuple is stable

Reply:
@DominiqueMakowski merge??? ^^

Reply:
✔️